src/common/library.c

```diff
@@ -63,21 +63,26 @@ void* Library_Sym(Library lib, const char* name)
 }
 
 #if PLAT_WINDOWS
+    #define LIB_PRE NULL
```

Comment:
If you can think of a more elegant way to prepend the `lib` feel free to suggest.